### ISSUE TITLE
Enhancement: In data, provide extra space between objects

### ISSUE THREAD

**Issue by djmtype** [estruyf/vscode-front-matter#719]
When specifying multiple objects within `frontMatter.data.files` array, it can be difficult to tell the objects apart. I'd treat the object titles either with a different font color and/or font size to help distinguish them apart.

At the very least, please provide extra margin between the objects. 

<img width="1270" alt="Screenshot 2023-12-05 at 3 04 27 PM" src="https://github.com/estruyf/vscode-front-matter/assets/930713/de006e9b-c01d-4d71-b543-ebafcf5b7400">

**Comment by estruyf** [comment]
Thanks @djmtype, for the suggestion!

**Comment by djmtype** [comment]
I think you want to target `.vscode-dark .autoform form > .fields > div > label:first-of-type`?

```
.autoform form > .fields > div > label:first-of-type {
  color: var(--primary-color);
}

.autoform form > .fields > div:not(:first-child) > label:first-of-type {
  margin-block-start: 2rem;
  border-block-start: 1px solid;
  padding-block-start: 2rem;
}

**Comment by estruyf** [comment]
@djmtype would you be able to provide a sample configuration so that I can reproduce it?

**Comment by estruyf** [comment]
Thanks @djmtype, I have included your fix into the 9.4.0 which should go out later today.
